### ISSUE TITLE
_reactNative.AccessibilityInfo.isScreenReaderEnabled is not a function. (followup on #1084)

### ISSUE THREAD

**Issue by eviliana** [wix/react-native-calendars#1234]
## Description

Issue from #1084 is not resolved by PR [ 7e0b1fc](https://github.com/wix/react-native-calendars/commit/7e0b1fcdcdf3e9ea51e603497ac574729b7f25d8) for RN versions < 0.59 because AccessibilityInfo's `isScreenReaderEnabled` is not supported by 0.59. Check the [official docs](https://reactnative.dev/docs/0.59/accessibilityinfo)

Backwards compatibility for AccessibilityInfo API should be applied.

The crash happens when trying to use ExpandableCalendar

## Expected Behavior

To add ExpandableCalendar and Timeline components and render without issues, as per demo

## Observed Behavior

- Update react-native-calendar from 1.265.0 to 1.300.0 in order to use new Timeline component.
- Insert code for use of ExpandableCalendar and Timeline components according to demos
- Crash with error `_reactNative.AccessibilityInfo.isScreenReaderEnabled is not a function.`

If there's an error message, please paste the *full terminal output and error message* in this code block:

```
ExceptionsManager.js:82 Unhandled JS Exception: TypeError: TypeError: _reactNative.AccessibilityInfo.isScreenReaderEnabled is not a function

This error is located at:
    in ExpandableCalendar (at asCalendarConsumer.js:13)
    in CalendarConsumer (at AppointmentsCalendar.js:211)
    in RCTView (at View.js:45)
    in View (at calendarProvider.js:169)
    in CalendarProvider (at AppointmentsCalendar.js:200)
    in AppointmentsCalendar (at AppointmentsScreen.js:377)
    in RCTView (at View.js:45)
    in View (at AppointmentsScreen.js:407)
    in AppointmentsScreen (at withDataLoader.js:343)
    in DataLoaderBase (created by Connect(DataLoaderBase))
    in Connect(DataLoaderBase) (created by Connect(Connect(DataLoaderBase)))
    in Connect(Connect(DataLoaderBase)) (created by SceneView)
    in SceneView (at StackViewLayout.js:793)
    in RCTView (at View.js:45)
    in View (at createAnimatedComponent.js:151)
    in AnimatedComponent (at StackViewCard.js:69)
    in RCTView (at View.js:45)
    in View (at createAnimatedComponent.js:151)
    in AnimatedComponent (at screens.native.js:59)
    in Screen (at StackViewCard.js:57)
    in Card (at createPointerEventsContainer.js:27)
    in Container (at StackViewLayout.js:858)
    in RCTView (at View.js:45)
    in View (at screens.native.js:83)
    in ScreenContainer (at StackViewLayout.js:311)
    in RCTView (at View.js:45)
    in View (at createAnimatedComponent.js:151)
    in AnimatedComponent (at StackViewLayout.js:307)
    in Handler (at StackViewLayout.js:300)
    in StackViewLayout (at withOrientation.js:30)
    in withOrientation (at StackView.js:79)
    in RCTView (at View.js:45)
    in View (at Transitioner.js:215)
    in Transitioner (at StackView.js:22)
    in StackView (created by Navigator)
    in Navigator (at createKeyboardAwareNavigator.js:12)
    in KeyboardAwareNavigator (created by SceneView)
    in SceneView (at createTabNavigator.js:39)
    in RCTView (at View.js:45)
    in View (at ResourceSavingScene.js:37)
    in RCTView (at View.js:45)
    in View (at ResourceSavingScene.js:26)
    in ResourceSavingScene (at createBottomTabNavigator.js:113)
    in RCTView (at View.js:45)
    in View (at screens.native.js:83)
    in ScreenContainer (at createBottomTabNavigator.js:103)
    in RCTView (at View.js:45)
    in View (at createBottomTabNavigator.js:102)
    in TabNavigationView (at createTabNavigator.js:197)
    in NavigationView (created by Navigator)
    in Navigator (created by SceneView)
    in SceneView (at StackViewLayout.js:793)
    in RCTView (at View.js:45)
    in View (at createAnimatedComponent.js:151)
    in AnimatedComponent (at StackViewCard.js:69)
    in RCTView (at View.js:45)
    in View (at createAnimatedComponent.js:151)
    in AnimatedComponent (at screens.native.js:59)
    in Screen (at StackViewCard.js:57)
    in Card (at createPointerEventsContainer.js:27)
    in Container (at StackViewLayout.js:858)
    in RCTView (at View.js:45)
    in View (at screens.native.js:83)
    in ScreenContainer (at StackViewLayout.js:311)
    in RCTView (at View.js:45)
    in View (at createAnimatedComponent.js:151)
    in AnimatedComponent (at StackViewLayout.js:307)
    in Handler (at StackViewLayout.js:300)
    in StackViewLayout (at withOrientation.js:30)
    in withOrientation (at StackView.js:79)
    in RCTView (at View.js:45)
    in View (at Transitioner.js:215)
    in Transitioner (at StackView.js:22)
    in StackView (created by Navigator)
    in Navigator (at createKeyboardAwareNavigator.js:12)
    in KeyboardAwareNavigator (at TabbarContainerScreen.js:47)
    in RCTView (at View.js:45)
    in View (at TabbarContainerScreen.js:46)
    in TabbarContainerScreen (created by Connect(TabbarContainerScreen))
    in Connect(TabbarContainerScreen) (created by SceneView)
    in SceneView (created by SwitchView)
    in SwitchView (created by Navigator)
    in Navigator (at createAppContainer.js:388)
    in NavigationContainer (at App.js:353)
    in RCTView (at View.js:45)
    in View (at App.js:352)
    in App (created by Connect(App))
    in Connect(App) (at Setup.js:101)
    in ThemeProvider (at Setup.js:100)
    in Provider (at Setup.js:99)
    in Root (at renderApplication.js:35)
    in RCTView (at View.js:45)
    in View (at AppContainer.js:98)
    in RCTView (at View.js:45)
    in View (at AppContainer.js:115)
    in AppContainer (at renderApplication.js:34)

This error is located at:
    in NavigationContainer (at App.js:353)
    in RCTView (at View.js:45)
    in View (at App.js:352)
    in App (created by Connect(App))
    in Connect(App) (at Setup.js:101)
    in ThemeProvider (at Setup.js:100)
    in Provider (at Setup.js:99)
    in Root (at renderApplication.js:35)
    in RCTView (at View.js:45)
    in View (at AppContainer.js:98)
    in RCTView (at View.js:45)
    in View (at AppContainer.js:115)
    in AppContainer (at renderApplication.js:34)
reactConsoleErrorHandler @ ExceptionsManager.js:82
console.error @ YellowBox.js:59
logToConsole @ RCTLog.js:47
logIfNoNativeHook @ RCTLog.js:30
__callFunction @ MessageQueue.js:366
(anonymous) @ MessageQueue.js:106
__guard @ MessageQueue.js:314
callFunctionReturnFlushedQueue @ MessageQueue.js:105
(anonymous) @ debuggerWorker.js:80

```

## Environment

Please run these commands in the project folder and fill in their results:

* `npm ls react-native-calendars`: react-native-calendars@1.300.0 
* `npm ls react-native`: 0.59.9

Also specify:

1. Phone/emulator/simulator & version: Tried on Android Nougat 7.0 (real device) and iPhone 11 (13.5) simulator

**Comment by emilisb** [comment]
Thanks for the issue, it should be fixed properly now.
